src/nvim/lua/treesitter.c

```diff
@@ -126,20 +126,22 @@ int tslua_add_language_from_object(lua_State *L)
 static const TSLanguage *load_language_from_object(lua_State *L, const char *path,
                                                    const char *lang_name, const char *symbol)
 {
+  char errbuf[IOSIZE];
   uv_lib_t lib;
   if (uv_dlopen(path, &lib)) {
+    xstrlcpy(errbuf, uv_dlerror(&lib), sizeof(errbuf));
```

Comment:
can't we use `IObuff`?
```suggestion
    xstrlcpy(IObuff, uv_dlerror(&lib), sizeof(IObuff));
```